imio/dms/mail/tests/test_wfadaptations.py

```
@@ -510,6 +512,50 @@ def test_OMToApproveAdaptationAfterNp1(self):
         self.common_tests()
 
+    def test_OMToApprove_with_n_plus_1_reapply(self):
+        """Test adaptations are reapplied from registry."""
+        self.portal.portal_setup.runImportStepFromProfile(
+            "profile-imio.dms.mail:singles", "imiodmsmail-om_n_plus_1_wfadaptation", run_dependencies=False
+        )
+        self.portal.portal_setup.runImportStepFromProfile(
+            "profile-imio.dms.mail:singles", "imiodmsmail-om_to_approve_wfadaptation", run_dependencies=False
+        )
+        self.assertTrue(load_workflow_from_package("outgoingmail_workflow", "imio.dms.mail:default"))
+        success, errors = apply_from_registry(reapply=True)
```

Ruff (0.15.17)

[warning] Unpacked variable `success` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)